CHANGES.rst

```diff
@@ -293,6 +293,14 @@ Other Changes and Additions
 4.0.1 (unreleased)
 ==================
 
+New Features
+------------
+
+astropy.utils
+^^^^^^^^^^^^^
+
+- ``astropy.utils.data.download_file` now supports FTPS/FTP over TLS. (#9964)
```

Comment:
Why is there a new feature entry? Also parenthesis should be square brackets.

Reply:
New feature entry because this really *is* a new feature of allowing FTPS for any use of `download_file`.  And that's actually kinda a big deal. It's just we're forced to do that to enable the bug fix, otherwise I would have had this be 4.1.

(parenths was a typo - fixed in the rebased version)

Reply:
Do we have a precedence to add "new feature" in a bug fix change log? I'll let @bsipocz decide.

Reply:
I think it's fine. It's not an API change, but non user oriented `utils` feature that helps to resolve a bug. 

We occasionally had `utils` new features in bugfix releases in the past, e.g. for the decorators. E.g. 2.0.12 or 1.3.1.